extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -43,50 +43,67 @@ void FlowFileRepository::flush() {
   auto batch = opendb->createWriteBatch();
   rocksdb::ReadOptions options;
 
-  std::vector<std::shared_ptr<FlowFile>> purgeList;
-
-  std::vector<rocksdb::Slice> keys;
-  std::list<std::string> keystrings;
-  std::vector<std::string> values;
+  std::list<ExpiredFlowFileInfo> flow_files;
 
   while (keys_to_delete.size_approx() > 0) {
```

Comment:
Please rename `keys_to_delete` to `keys_to_delete_` to show membership

Reply:
done